### PR TITLE
New Dresses and Holiday Clothing

### PR DESCRIPTION
**Pull request by jan-antilles**
[feature][sprites][add to wiki]

## About the PR 
This PR adds the following new clothing items:

-Sleeveless Dress in Red/Blue/Green/White/Black
     - These are in the Clothing Booth under the "Dresses" category
-Sleeveless Dress in Santa and Snowy
     - These are in the Clothing Booth under the new "Holiday" category
-Green Waistcoat
     - This is in the Clothing Booth under the new "Holiday" category
-Hairbows in Cranberry, Dark Green, and Royal Blue
     - These are in the Clothing Booth under the "Accessories" category
-Holly Barrettes and Holly Top Hat
     -  In the Clothing Booth in the new "Holiday" category

image
image
image

In addition, the three Sweater designs (Reindeer, Diamonds, and Snowflake) have been added to the "Holiday" category in the clothing booth.

## Why's this needed? 
These are cute, inexpensive new dresses and holiday clothing items that will give players additional fun customization options. The new category on the clothing booth will keep the holiday clothing confined to a single category while leaving the non-seasonal items easy to find in the rest of the established categories.

## Changelog

```changelog
(u)Jan.antilles
(*)Added new dresses and holiday clothing, all available from the Clothing Booth!
```

### CODE REVIEW

File: code/obj/item/clothing/hats.dm [hunk 1]
```diff
@@ -1489,6 +1489,10 @@ ABSTRACT_TYPE(/obj/item/clothing/head/barrette)
 		name = "gold barrettes"
 		icon_state = "barrette-gold"
 		item_state = "barrette-gold"
+	holly
+		name = "holly barrette"
```
Comment (Zonespace27) [inline code/obj/item/clothing/hats.dm:1493]:
```suggestion
		name = "holly barrettes"
```
consistency

**Comment by jan-antilles** [comment]
updated description with the Green Waistcoat and the Holly Barrettes change

**Review by pali6 — APPROVED** [review]
I feel like some of the sprites could have a tiny bit more texture to them but also I'm not a sprite / art person so who knows. Looks good to me. There has been some internal discussion about how we'd like new gimmick clothing like this to be introduced and I'm unsure if it might be preferrable to instead have it be a QM purchase or sourced some other way.

**Comment by pali6** [comment]
It has been requested by other admins that the Spacemas section be in an `#ifdef XMAS`.
It has also been mentioned that formal dresses don't fit into ss13, especially not as every round free customization from the clothing booth.

**Comment by jan-antilles** [comment]
Those changes make sense to me! Let me go figure out how to make them happen!

**Comment by jan-antilles** [comment]
Those changes make a lot of sense to me, especially since the fancy furniture is similarly locked! 

I think for now I'm going to hold off on this rather than trying to rush it for the event, and add some more texture and detail to the dresses so they'll fit well in a Fancy Apparel crate. :)

Thank you!

**Pull request closed without merging**